tests/Support/SupportUriTest.php

```diff
@@ -220,4 +220,15 @@ public function test_path_segments()
 
         $this->assertEquals(3, $uri->pathSegments()->count());
     }
+
+    public function test_macroable()
+    {
+        Uri::macro('firstSegmentMacro', function () {
```

Comment:
Used `Macro` suffix here, just in case a `firstSegment` method will be added to the class in the future.